### PR TITLE
add angular-ui-codemirror@0.3.0 w/ git auto-update

### LINKED ISSUE #10032: [Request] angular-ui/ui-codemirror
**Library name:** angular-ui/ui-codemirror
**Git repository url:** https://github.com/angular-ui/ui-codemirror
**npm package url(optional):**  https://www.npmjs.com/package/angular-ui-codemirror
**License(s):** MIT
**Official homepage:** https://angular-ui.github.io/ui-codemirror/

### PR DESCRIPTION
**Pull request by lcd78706**
close #10032, cc @douglasduteil

Pull request for issue: #10032 
Related issue(s): # #

@x09326  Please review this PR. Thanks!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/angular-ui/ui-codemirror
 * Official website (optional, not the repository):  https://angular-ui.github.io/ui-codemirror/
 * NPM package url (optional): https://www.npmjs.com/package/angular-ui-codemirror
 * License and its reference: MIT https://github.com/angular-ui/ui-codemirror/commit/7b0f5220e7771fd36e5fae9d8814250b9cf95952#diff-0a08a7565aba4405282251491979bb6bR6
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 324
   - Count of watchers: 33
   - Count of forks: 147
 * NPM download stats (optional):
   - Downloads in the last day: 169
   - Downloads in the last week: 1535
   - Downloads in the last month: 8635

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/angular-ui-codemirror/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "url": "git://github.com/angular-ui/ui-codemirror.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/angular-ui/ui-codemirror.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "**/*"
```
Comment (x09326) [inline ajax/libs/angular-ui-codemirror/package.json]: @lcd78706 
The `fileMap` is wrong, you need to see the file structure of the version branch like **v0.3.0**, so the `basePath` should be `""` and the `files` should be `ui-codemirror*`.

Comment (lcd78706) [inline ajax/libs/angular-ui-codemirror/package.json]: @x09326 I have modified it.

**Comment by x09326** [comment]
LGTM
@cdnjs/intern2 please help me review it, thanks.

File: ajax/libs/angular-ui-codemirror/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "angular-ui-codemirror",
+  "filename": "ui-codemirror.min.js",
+  "version": "0.3.0",
+  "description": "This directive allows you to add CodeMirror to your textarea elements.",
+  "author": "https://github.com/angular-ui/ui-codemirror/contributors",
+  "license": "MIT",
+  "homepage": "https://angular-ui.github.com",
```
Comment (kennynaoh) [inline ajax/libs/angular-ui-codemirror/package.json]: @lcd78706 I think this is much better https://angular-ui.github.io/ui-codemirror/

**Comment by kennynaoh** [comment]
ping @x09326

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Comment by x09326** [comment]
@lcd78706 
Please rebase to the latest master branch, thanks.

**Comment by lcd78706** [comment]
@x09326  I have done it.

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Pull request merged**